### PR TITLE
link update

### LINKED ISSUE #303: Link for debugging Doc for e2e tests not working.
The link is expired, It needs to be updated to `https://github.com/oppia/oppia/wiki/Debug-end-to-end-tests`
Currently it is taking to home page.
Screenshot 2024-04-04 at 8 14 53 AM

### PR DESCRIPTION
**Pull request by rahat2134**
Fix #303

### CODE REVIEW

**Comment by rahat2134** [comment]
@seanlip PTAL

**Comment by rahat2134** [comment]
@seanlip PTAL.

File: End-to-End-Tests.md [hunk 1]
```diff
@@ -307,7 +307,7 @@ If you need to, you can add a new test suite to [`core/tests/webdriverio_desktop
 
 3. Write the tests! Each test should step through one of your user journeys, asserting that the page is in the expected state along the way.
 
-For information on writing tests with webdriverio, see the [webdriverio's documentation](https://webdriver.io/docs/gettingstarted). If you need to work out why your tests aren't working, check out our [[debugging guide for E2E tests|Debug-end-to-end-tests-webdriverio]].
+For information on writing tests with webdriverio, see the [webdriverio's documentation](https://webdriver.io/docs/gettingstarted). If you need to work out why your tests aren't working, check out our [debugging guide for E2E tests](https://github.com/oppia/oppia/wiki/Debug-end-to-end-tests).
```
Comment (seanlip) [inline End-to-End-Tests.md]: Thanks @rahat2134. Could you use the existing format: [[debugging guide for E2E tests|Debug-end-to-end-tests]] ?

**Review by seanlip — CHANGES_REQUESTED** [review]
Hi @rahat2134, thanks, just one comment!

File: End-to-End-Tests.md (hunk 1 above)
Comment (rahat2134) [reply End-to-End-Tests.md]: Done

Comment (seanlip) [reply End-to-End-Tests.md]: No, you haven't done what I asked. For internal wiki link pages, you don't need the github.com/oppia/oppia/wiki/ prefix. (Have a look at what it was before.)

Comment (rahat2134) [reply End-to-End-Tests.md]: @seanlip  sorry! Done Now. 
`[[debugging guide for E2E tests|Debug-end-to-end-tests]]`

**Review by seanlip — APPROVED** [review]
Thanks!

**Pull request merged**